estate/models/estate_property_offer.py

```diff
@@ -59,11 +59,16 @@ def action_refuse_offer(self):
         self.write({"status": "refused"})
         return True
 
-    @api.model
-    def create(self, vals):
-        actual_property = self.env["estate.property"].browse(vals["property_id"])
-        max_price = max(actual_property.offer_ids.mapped("price"))
-        if max_price and float_compare(vals["price"], max_price, precision_digits=2) == -1:
-            raise UserError("The offer price cannot be lower than %.2f" % max_price)
-        actual_property.status = "offer_received"
-        return super().create(vals)
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            actual_property = self.env["estate.property"].browse(vals["property_id"])
+            offers = actual_property.offer_ids.mapped("price")
+            if len(offers) > 0:
+                max_price = max(offers)
+            else:
+                max_price = 0
```

Comment:
The variable `offers` actually is not offers but offer prices, please use meaningful names. And you don't need to do a `len()`  you could do this:

```suggestion
            offer_prices = actual_property.offer_ids.mapped("price")
            if offer_prices:
                max_price = max(offer_prices)
            else:
                max_price = 0
```
Please try this.

Reply:
I tried it but the result for offers_price using mapped is an empty list, that's why I used len() but now I changed it and I'm not using that method.